### DISCUSSION TITLE
CDN Link Not Work With Me When I Add It To an HTML file.

### DISCUSSION THREAD

**Discussion by minaragy** [twbs/bootstrap#39143] [Q&A] [answered]
CDN Link Not Work With Me When I Add It To an HTML file. 
While I make all steps right Can Anyone Help Me?

**Comment by julien-deramond** [comment]
If you are following the integration described in https://getbootstrap.com/docs/5.3/getting-started/download/#cdn-via-jsdelivr, you should be good to go.
I can see that you are based in Egypt, so maybe your issue is linked to https://github.com/orgs/twbs/discussions/38782 where, apparently, some Egyptian ISPs are blocking the `cdn.jsdelivr.net` domain.

**Comment by minaragy** [comment]
Thanks for the clarification

There is no solution for any Egyptian programmer to use Bootstrap right?

**Reply by julien-deramond** [reply, marked as answer]
I'd say it's safe to use `unpkg.com` instead of `cdn.jsdelivr.net`.
For example, https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css becomes https://unpkg.com/bootstrap@5.3.1/dist/css/bootstrap.min.css (`integrity` sha384 value stays the same)
Don't hesitate to tell me if it works better.
If so, I'll try to create a PR to talk about this workaround in our documentation.

**Reply by minaragy** [reply]
Yes, it Works Thank You Very Much. I have 3 days searching for a solution.
Thank You Dear
